frontend/src/pages/home/videos/sections/research/ScientificLiteratureBox.tsx

```diff
@@ -0,0 +1,215 @@
+    {
+      name: 'Science Communication Desperately Needs More Aligned Recommendation Algorithms.',
+      authors: 'Lê Nguyên Hoang (2020). Frontiers Communications.',
+      url: 'https://www.frontiersin.org/articles/10.3389/fcomm.2020.598454/full',
+      icon: Campaign,
+    },
+  ];
+
+  const studentWorks = [
+    {
+      name: 'Active learning for video recommendation system.',
+      authors: "Marc Gay-Balmaz, EPFL Master's Thesis (2022).",
+      url: 'https://www.dropbox.com/sh/kztfgpz0epfr460/AACO-37hc-KdZmgfxvsxHmjLa/Active%20Learning%20%28Marc%20Gay-Balmaz%29.pdf',
+      icon: School,
+    },
+    {
+      name:
+        'Byzantine-resilient account validation through vouching : research and application to' +
+        ' Tournesol project.',
+      authors: "Bérangère Colbois. EPFL Master's semester project (2022).",
+      url: 'https://www.dropbox.com/sh/kztfgpz0epfr460/AAA6m1mb4OfmiOZ3wUS8s4dLa/Bayesian%20Byzantine%20voting%20rights%20%28Mac%20Vandelle%29.pdf',
+      icon: School,
+    },
+    {
+      name: 'Bayesian Byzantine Resistance.',
+      authors: "Marc Vandelle. EPFL Master's semester project (2022).",
+      url: 'https://www.dropbox.com/sh/kztfgpz0epfr460/AAAXHAWRGKh3MWXhfSDUkVkXa/Vouching%20%28B%C3%A9rang%C3%A8re%20Colbois%29.pdf',
+      icon: School,
+    },
+  ];
+
+  return (
+    <Paper>
+      <Box
+        p={2}
+        color="#fff"
+        bgcolor="#1282B2"
+        sx={{ borderTopLeftRadius: 'inherit', borderTopRightRadius: 'inherit' }}
+      >
+        <Typography variant="h4">
+          {t('scientificLiteratureBox.scientificLiterature')}
+        </Typography>
+      </Box>
+      <Box>
+        <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
+          <Tabs
+            value={selectedTab}
+            onChange={handleTabChange}
+            aria-label="basic tabs example"
+          >
+            <Tab
+              label={t('scientificLiteratureBox.chosenArticles')}
+              {...a11yProps(0)}
+            />
+            <Tab
+              label={t('scientificLiteratureBox.studenWorks')}
+              {...a11yProps(1)}
+            />
+          </Tabs>
+        </Box>
+        <TabPanel value={selectedTab} index={0}>
+          <List>
+            {chosenArticles.map((article, idx) => (
+              <React.Fragment key={article.name.replace(' ', '_')}>
+                <ListItem>
+                  <ListItemAvatar>
+                    <Avatar>
+                      <article.icon />
+                    </Avatar>
+                  </ListItemAvatar>
+                  <ListItemButton
+                    href={article.url}
+                    target="_blank"
+                    rel="noopener"
+                    disableGutters
+                  >
+                    <ListItemText
+                      primary={article.name}
+                      secondary={article.authors}
+                    />
+                  </ListItemButton>
+                </ListItem>
```

Comment:
It would be slightly more aesthetically pleasing to include the avatar inside the button, and keep the button gutters by removing the `ListItem` padding.

```suggestion
                <ListItem disablePadding>
                  <ListItemButton
                    href={article.url}
                    target="_blank"
                    rel="noopener"
                  >
                    <ListItemAvatar>
                      <Avatar>
                        <article.icon />
                      </Avatar>
                    </ListItemAvatar>
                    <ListItemText
                      primary={article.name}
                      secondary={article.authors}
                    />
                  </ListItemButton>
                </ListItem>
```

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1202/commits/5d34c4ab918f0f0b5cd90c93799b312c59611f2b